src/js/form.js

```diff
@@ -13,6 +13,11 @@ const createTitle = () => {
 }
 // createElement('div', { className: 'form-group' })
 
+const getCurrentTime = () => {
+  const date = new Date();
+  return date.toLocaleTimeString('fr-FR', { hour: '2-digit', minute: '2-digit' });
+}
+
```

Comment:
This should be removed if https://github.com/LAB-MI/attestation-deplacement-derogatoire-q4-2020/pull/91 is accepted.